tests/filecheck/parser-printer/float_parsing.mlir

```diff
@@ -11,12 +11,12 @@
   "test.op"() {"value" = 34.e0 : f32} : () -> ()
   // CHECK-NEXT: "test.op"() {"value" = 3.400000e+01 : f32} : () -> ()
 
-  "test.op"() {"value" = 34.e-23 : !f32} : () -> ()
+  "test.op"() {"value" = 34.e-23 : f32} : () -> ()
```

Comment:
I couldn't find any MLIR reference where `!f32` is a valid type for this built-in.
Not sure if I'm overlooking something.